microcosm_daemon/daemon.py

```diff
@@ -141,7 +143,7 @@ def initialize(self):
         # reprocess the arguments because some aspects of argparse are not pickleable
         # and will fail under multiprocessing
         self.parser = self.make_arg_parser()
-        self.args = self.parser.parse_args()
+        self.args, _ = self.parser.parse_known_args()
```

Comment:
This is to avoid errors when running a daemon with multiple processes in tests (the arguments to `nosetests` would end up being passed down here in the subprocesses).
Note that it doesn't make any difference in practice – `parse_args` is run on the main process with the same args parser, so any unknown argument would have errored out already.